cniconf/watcher.go

```diff
@@ -78,7 +83,39 @@ func (w *watcher) onChange(version string) error {
 	return nil
 }
 
-func (w *watcher) apply(network metadata.Network) error {
+func (w *watcher) inserthostSubnetRules(host metadata.Host, content []byte) ([]byte, error) {
+	subnet, _ := host.Labels[hostSubnetLabel]
```

Comment:
@niusmallnan Let's go with option 1. So that in future if we want to support per_host_subnet on ipsec/vxlan we can support that too.